src/ingest_llm_as/parsers/python_ast_parser.py

```
@@ -292,69 +301,69 @@ def _create_code_element(
         signature = f"{node.name}({', '.join(args)})"
-        
+
         # Return annotation
         if node.returns:
             signature += f" -> {ast.unparse(node.returns)}"
-        
+
         return signature
-    
+
     def _is_method(self, node: ast.FunctionDef) -> bool:
```

Ruff (0.13.3)

Unused method argument: `node`

(ARG002)